### ISSUE TITLE
Connection not working

### ISSUE THREAD

**Issue by virgileMaon** [clariusdev/cast#148]
Hello, 

From today I can't connect to clarius via Cast api.
I have this error message : 
"software revisions are different, the connection may not operate as intended"

here some information: 
Cast api gives me : 
Firmware V1: 11.0.0-540+bf1bbd364b
Firmware HD: 11.0.0-525+bf1bbd364b
Firmware HD3: 11.0.0-525+bf1bbd364b

Clarius app gives me : 
Version 11.1.0-444+d52e601542
Version scanner software 11.1.0-554+752c0d8892
Vesion scanner software HD 11.1.0-540+752c0d8892

How can I fixe this?

Best regard
virgile.dupuis@deski.io

**Comment by clariusk** [comment]
you will still be able to image and work with the API, as we did not change anything to our protocol between versions. In the meantime, we'll be uploading API version 11.1 soon, thanks

**Comment by virgileMaon** [comment]
Yes it's true but it's problematique trigger an error while it's just an information.

**Comment by clariusk** [comment]
please find the latest release here: https://github.com/clariusdev/cast/releases
